lib/Config.php

```diff
@@ -132,6 +132,10 @@ public function canUserEnableSIP(IUser $user): bool {
 		return $this->canEnableSIP[$user->getUID()];
 	}
 
+	public function isBreakoutRoomsEnabled(): bool {
+		return $this->config->getAppValue('spreed', 'call_recording', 'yes') === 'yes';
```

Comment:
1. Wrong function name
2. Should check for external signaling

```suggestion
	public function isRecordingEnabled(): bool {
		return $this->getSignalingMode() !== self::SIGNALING_INTERNAL && $this->config->getAppValue('spreed', 'call_recording', 'yes') === 'yes';
```

Reply:
copy past.

Reply:
I fixed in second PR and forgot to fix here.